lib/plugin.js

```diff
@@ -21,7 +24,18 @@ function plugin (pluginOptions) {
       return next(new Error('missing shared secret'))
     }
 
-    return next(new createError[options.verificationError](options.verificationErrorMessage))
+    try {
+      const parsedSignatureHeader = parseSignatureHeader(req)
+      extractSignature(parsedSignatureHeader)
+    } catch (error) {
+      console.log(error)
```

Comment:
Rather than console.logs, we can use Fastifys logger here.

https://www.fastify.io/docs/v2.0.x/Logging/

So this could be: `request.log.error(error)`. This is, if we want to log here.